pallets/parachain-staking/src/set.rs

```diff
@@ -297,4 +328,57 @@ mod tests {
 		set.clear();
 		assert_eq!(set, OrderedSet::new());
 	}
+
+	#[test]
+	fn linear_search() {
```

Comment:
I would rather test `linear_search` directly. Looks odd to test `linear_search` and the test doesn't contain a single reference to `linear_search` 😜 .

Reply:
True. I renamed it and added specific tests for `linear_search` in e164d7d.